ivi-layermanagement-api/ilmControl/src/ilm_control_wayland_platform.c

```diff
@@ -197,7 +197,9 @@ static struct wl_output_listener output_listener = {
     output_listener_geometry,
     output_listener_mode,
     output_listener_done,
-    output_listener_scale
+    output_listener_scale,
+    NULL,
```

Comment:
we should define an empty function instead of a NULL function pointer! for all NULL here!

Reply:
Hello @efriedrich I updated follow your comment. Please help me review again. thanks